people/the8472.toml

```
@@ -0,0 +1,4 @@
+name = 'the8472'
+github = 'the8472'
+github-id = 1065730
+email = 'git@infinite-source.de'
```

```suggestion
email = 'the8472.rs@infinite-source.de'
```

Makes it easier to replace in case it gets spammed.